tutorial.ipynb

```diff
@@ -24,9 +24,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
+    "os.environ[\"MUJOCO_GL\"] = os.environ.get(\"MUJOCO_GL\", \"egl\")\n",
```

Comment:
Maybe add comment here to try "glfw" or "osmesa" if egl throws errors (think "glfw" was the most robust pick for us)

Reply:
I removed the lines for consistency with the basic tutorials (i.e. keep this setup on the user side)